### ISSUE TITLE
RODA-in fails to validate metadata against its schema (if schema includes other schemas) in Windows

### ISSUE THREAD

**Issue by hsilva-keep** [keeps/roda-in#290]
Most certainly related to org.roda.rodain.core.ControllerUtils.validateSchemaWithoutCatch(String, InputStream) line
```
StreamSource streamSource = new StreamSource(schemaStream, ConfigurationManager.getRodainPath().toString());
```
and org.roda.rodain.core.utils.ResourceResolver.

**Comment by hsilva-keep** [comment]
And having non-ASCII chars in the windows user also can be the cause.

**Comment by jmaferreira** [comment]
@chalkos test this extensibly on Windows please.

**Comment by hsilva-keep** [comment]
This has been solved (5509966aa31843b0f2f88750b3283329a75d593b, 682e3c2bfae54cc1e4d84453a819049c8201be96 & 7db2009dd9192a4f03b6288f9002aa27a3f69edd) & tested by me in both ubuntu, mac os & windows (both jar & exe).
